### PR TITLE
Cheery-pick bugs fix to Release 1.5.0

### PR DESCRIPTION
**Pull request by ninjadq**
1. Fix bug 4665 project admin cannot change the role of some user.
2. Add query string when getting members
3. ix UI/UX issues and manually refresh component when fresh
4. Fix configure save err #4698
5. Fix readonly mode show in vic and rewording the tooltip
6. Fix default view problem and styles issues

### CODE REVIEW

File: src/ui_ng/src/app/config/config.component.ts [hunk 1]
```diff
@@ -266,7 +266,7 @@ export class ConfigurationComponent implements OnInit, OnDestroy {
                         this.msgHandler.handleReadOnly();
                     }
 
-                    if (changes['read_only'].toString() === "false") {
+                    if (changes['read_only'] && changes['read_only'].toString() === "false") {
```
Comment (steven-zou) [inline src/ui_ng/src/app/config/config.component.ts]: Incorrect statement logic

Comment (ninjadq) [reply src/ui_ng/src/app/config/config.component.ts]: 'changes' only contains number and string. so there are no boolean problems.

**Comment by reasonerjt** [comment]
Please rebase

**Comment by pengpengshui** [comment]
It’s dq’s commits

**Pull request merged**